src/config.js

```diff
@@ -1908,10 +1968,10 @@ export const CONFIG = {
       platform: 'windows10-64',
       benchmarks: DEFAULT_SUITES,
     },
-    win10Laptops: {
-      label: 'Windows 10 64bit (laptops)',
-      platform: 'windows10-64-ux',
-      benchmarks: DEFAULT_SUITES,
+    win10ref2017: {
+      label: 'Windows 10 64bit (2017 reference laptop)',
+      platform: 'windows10-64-ref-hw-2017',
+      benchmarks: WIN10_REF_2017,
```

Comment:
Why a different set of benchmarks for the reference laptops?

Reply:
I concatenated the newly discovered tests with the DEFAULT_SUITES

Reply:
Thanks. Are these new tests unique to this platform, or could we just add them to DEFAULT_SITES?

Reply:
If you search for one of them:
https://searchfox.org/mozilla-central/search?q=rasterflood_gradient&path=
you will get to:
https://searchfox.org/mozilla-central/source/testing/talos/talos.json#37
if you look for `g4`:
https://treeherder.mozilla.org/#/jobs?repo=mozilla-central&searchStr=g4
you will see that it most likely applies to all.

Can I assume r+ after I add it to DEFAULT_SUITES and I merge this w/o another review?

Reply:
Yes. I'll give the r+ now